### PR TITLE
Splitting species changes out of /decl/species PR.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
Going to split stuff out until it's just the /decl changes and hopefully isolate where the unit tests start failing.

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
Going to need to test a bunch of this before a merge.

**Review by comma — COMMENTED** [review]
so here we compare result of that proc to species
```
if(H.species.get_root_species_name() != SPECIES_MANTID_ALATE)
```
but here to bodytypes 
```
        var/bodytype = user.species.get_root_species_name(user)
        if(bodytype in list(BODYTYPE_MANTID_SMALL, BODYTYPE_MANTID_LARGE, BODYTYPE_SNAKE))
```
whch one is real

File: code/modules/species/species_shapeshift.dm [hunk 1]
```diff
@@ -13,57 +13,64 @@ var/list/wrapped_species_by_ref = list()
 
 	var/list/valid_transform_species = list()
 	var/monochromatic
-	var/default_form = SPECIES_HUMAN
+	var/default_form
+
+/datum/species/shapeshifter/New()
+	default_form = GLOB.using_map.default_species
+	valid_transform_species |= default_form
+	..()
```
Comment (comma) [inline code/modules/species/species_shapeshift.dm:22]: shapeshifters don't fake their root_species_name is that intended?

Comment (MistakeNot4892) [reply code/modules/species/species_shapeshift.dm:22]: Fixed.

**Comment by MistakeNot4892** [comment]
Bodytype was the intended call, fixed.

**Review by comma — DISMISSED** [review]

**Review by comma — APPROVED** [review]

**Pull request merged**